### ISSUE TITLE
Blog for Origen?

### ISSUE THREAD

**Issue by welguisz** [Origen-SDK/origen#205]
Should we start a blog on Origen showing how versatile Origen is in the Semiconductor Space?

- Test Patterns
- Documentation
- Information Flow and Data Flow

**Comment by info-rchitect** [comment]
@welguisz i agree.  I have written about 4-5 blogs per month internally that have generally been well received.  I think 'vlogging' is also a good idea, similar to @ginty 's videos.

**Comment by ginty** [comment]
Yes, we definitely should.

We already have this forum setup which would be a good place to write them I think, though I would be open to other suggestions if anyone has any: https://disqus.com/home/channel/origensdk/

Definitely it should be somewhere outside of origen-sdk.org though so that our search engine ranking can benefit from the incoming 3rd party links to the relevant content on origen-sdk.org.

In terms of promotion, this seems like a good place to post links to any new blog posts: https://www.linkedin.com/groups/941397
